.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,20 @@
+
+**What you expected**
+
+**Environment**
+- `container --version`:
+- macOS version:
+- bushel version:
+
+**Message log**
+Relevant lines from bushel's message log (`m`), which preserves full stderr:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)